README.md

```diff
@@ -615,5 +679,7 @@ versions as the required `pip` versions.
 [Matrix Space]: https://matrix.to/#/%23jazzband:matrix.org
 [pip-tools-overview]: https://github.com/jazzband/pip-tools/raw/main/img/pip-tools-overview.svg
 [environment-markers]: https://peps.python.org/pep-0508/#environment-markers
+[pep-517]: https://peps.python.org/pep-0517/
+[pep-660]: https://peps.python.org/pep-0660/
```

Comment:
No need to make separate labels for these links:
```suggestion
[PEP 517]: https://peps.python.org/pep-0517/
[PEP 660]: https://peps.python.org/pep-0660/
```

Reply:
@merwok yes, one of those declares the `pep` role. And it'd work, if we were to only modify content under `docs/` — MyST picks it up just fine. But since this is in the repo-root README, GitHub also renders this file and it would only render GitHub-flavored Markdown. So we're limited to that for files that we want GitHub to also render correctly.
Well, not just GitHub but PyPI too as it's also included in the package distribution metadata.

Reply:
Whenever we move things from README to other Sphinx-land documents, it'll be possible to use MyST syntax for roles for those moved-out parts.

Reply:
Oops I missed that this was markdown, the underpowered competitor to reST 😛

Reply:
@merwok GitHub uses GFM — mostly native Markdown, but Sphinx uses MyST which is a superset of Markdown that provides features that were previously only accessible via RST. You can actually use roles and directives in MyST-flavored Markdown but the problem here is that this specific file is being processed by both parsers in different places.